### DISCUSSION TITLE
希望自定义顶栏的设置项也加入到首页的功能选单

### DISCUSSION THREAD

**Discussion by Kosette** [the1812/Bilibili-Evolved#2666] [功能建议 / Ideas]
从预览版切换到正式版，重新设置了一下，一时间没找到自定义顶栏的入口，建议和简化首页一起放在功能选单里面。

**Comment by the1812** [comment]
这关简化首页什么事, 自定义顶栏的范围可不止首页

**Reply by Kosette** [reply]
我的意思是在功能选单里面加入一个入口，至于是不是只在主页显示不是重点。就是下面这个入口，毕竟是是和简化主页一样是样式设置，这样直观一点。
![image](https://user-images.githubusercontent.com/35268640/145699466-dc24e9e8-edc0-445b-9d08-386df4bfcc8b.png)

**Reply by the1812** [reply]
可以的

**Reply by Kosette** [reply]
辛苦👍
